.idea/workspace.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be removed and `.idea` folder should be in .gitignore

Reply:
I agree, will remove it.